eng/pipelines/build-all-lib.yml

```diff
@@ -194,10 +194,10 @@ extends:
 
         # Index sources and publish symbols
 
-        - task: PublishSymbols@2
-          inputs:
-            SearchPattern: '**/bin/**/*.pdb' # string. Required. Search pattern. Default: **/bin/**/*.pdb.
-            SymbolServerType: 'TeamServices'
+        #- task: PublishSymbols@2
```

Comment:
Why did you comment these lines?

Reply:
I did that earlier. Probably got pulled in because of rebase